src/Components/MovieTile/MovieTile.module.css

```
@@ -0,0 +1,48 @@
+.container {
+  width: 250px;
+  color: white;
+  background-color: transparent;
+  overflow: hidden;
+  font-weight: 100;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
+    'Droid Sans', 'Helvetica Neue', sans-serif;
+}
+
+.container img {
```

Selectors should be as simple as possible.
Usually, a class name is set to everything:

```suggestion
.image {
```

Only if there is some complex logic, complex selectors are used.